src/utils/updateDatabase.ts

```diff
@@ -46,3 +31,48 @@ export async function updateDatabase(collections: Collection[]) {
     /* tslint:enable:no-unused-expression */
   }
 }
+
+const upsert_data = async (collection, collections) => {
+  try {
+    for (const entry of collections) {
+      // if an entry's `price_guidance` is left null in the dataset, null
+      // it out in the database - after this is finished we will compute price
+      // guidance for all collections that have a null value for it.
+      if (!entry.price_guidance) {
+        extend(entry, { price_guidance: null })
+      }
+      await collection.update({ slug: entry.slug }, entry, { upsert: true })
+    }
+  } catch (e) {
+    console.log("Error upserting data!")
+    console.log(e.message)
+    throw e
+  }
+}
+
+const update_price_guidance = async collection => {
```

Comment:
Any reason these two new functions aren't 🐪 ? (I'd prefer to be consistent in at least the cases where we _do_ have control)